apps/expo-go/ios/Exponent/Kernel/AppLoader/EXAbstractLoader.h

```diff
@@ -50,6 +50,7 @@ typedef enum EXAppLoaderRemoteUpdateStatus {
 @property (nonatomic, readonly) EXAppLoaderRemoteUpdateStatus remoteUpdateStatus;
 @property (nonatomic, readonly) BOOL shouldShowRemoteUpdateStatus;
 @property (nonatomic, readonly) BOOL isUpToDate;
+@property (nonatomic, readonly) NSNumber * _Nullable launchDuration;
```

Comment:
```suggestion
@property (nonatomic, readonly, nullable) NSNumber *launchDuration;
```

maybe it's better to put nullable here